src/auth.rs

```diff
@@ -1,3 +1,33 @@
+//! Client Authentication.
+//!
+//! Some RPC nodes will require authentication before requests can be sent to them.
+//!
+//! This is done by adding your api key to the request as a header.
```

Comment:
Reference the [`JsonRpcClient::header`](https://docs.rs/near-jsonrpc-client/latest/near_jsonrpc_client/struct.JsonRpcClient.html#method.header) method.

```suggestion
//! This is done by adding your api key to the request as a [`header`](crate::JsonRpcClient::header).
```